src/components/Form/Buttons/BasicAcceptButton.tsx

```diff
@@ -0,0 +1,32 @@
+import { Button } from '@material-ui/core';
+import clsx from 'clsx';
+import React from 'react';
+import { useStyles } from './styles/BasicButton.style';
+
+interface IBasicAcceptButtonProps {
+  className?: string;
+  disabled?: boolean;
+  label?: string;
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+export const BasicAcceptButton: React.FC<IBasicAcceptButtonProps> = ({
```

Comment:
this and BasicSignButton seem to be 100% the same. The only difference is styles. I think they can be merged into one